docs/user/howto/lintchart.md

```diff
@@ -24,7 +24,7 @@ $ hypper lint cmd/hypper/testdata/testcharts/hypper-annot
 While running it against a vanilla Helm chart with no extra Hypper annotations will emit several `[WARNING]`, recommending to set certain values that Hypper supports:
 
 
-```shell
+```terminal
```

Comment:
same here

Reply:
resolving, let's leave the other conversation.